### PR TITLE
Tom dem 31 integrate

### PR DESCRIPTION
**Pull request by tladendo**
fyi @indytechcook @vietnogi -- here's the integration of most of the wireframe templates

### CODE REVIEW

File: src/client/app/submit-work/features/scripts/features.controller.js [hunk 1]
```diff
@@ -19,9 +19,11 @@
     function activate() {
       logger.info('Activated Features View');
       vm.work = SubmitWorkService.getCurrent();
-      FeatureService.getFeatures().then(function(features) {
-        vm.work.features = features;
-      });
+      if (vm.work.features.length == 0) {
```
Comment (indytechcook) [inline src/client/app/submit-work/features/scripts/features.controller.js]: We probably don't want to assign all of the features to the workrequest object.  They only should be part of the object if the user select them.  The FeatureService was just to get the metadata/list of features and not which features are assigned to the work request object.

I apologize for sending you down this initial path.

**Comment by indytechcook** [comment]
Made one comment, otherwise lgtm.

File: src/client/app/submit-work/features/scripts/features.controller.js (hunk 1 above)
Comment (tladendo) [inline src/client/app/submit-work/features/scripts/features.controller.js]: I'm going to handle this at the service level by checking which fields are selected. I think we can keep this how it is.

**Comment by vietnogi** [comment]
lgtm

**Pull request merged**